SPECS/helm/helm.spec

```diff
@@ -51,7 +51,14 @@ install -m 755 ./helm %{buildroot}%{_bindir}
 %doc ADOPTERS.md SECURITY.md code-of-conduct.md CONTRIBUTING.md README.md
 %{_bindir}/helm
 
+
+%check
+go test -v ./cmd/helm
+
 %changelog
+* Wed Dec 21 2022 Jon Slobodzian <joslobo@microsoft.com> - 3.4.1-14
+- Patch CVE-2022-23525, CVE-2022-23526
```

Comment:
Can you add a note about enabling the check tests as well

Reply:
Fixed